config/profile-partner-1.0.yaml

```diff
@@ -28,4 +28,6 @@ checks:
     - name: v1.0/images-are-certified
       type: Mandatory
     - name: v1.0/chart-testing
-      type: Mandatory
\ No newline at end of file
+      type: Mandatory
```

Comment:
If we are adding a new mandatory test we need to create a new profile-partner-1.1.yaml. And same for redhat and community.